tilt_devnet/docker_images/Dockerfile.lerna

```diff
@@ -0,0 +1,35 @@
+# The aim for the base image here is to only keep package*.json files while having the
+# same directory structure to be able to have lerna dependencies installed once and cached
+# as long as the package*.json files have not changed. In the future, we can further optimize
+# it by creating slim package files that only contain *dependencies and name keys.
+
+FROM node:18.13.0@sha256:d9061fd0205c20cd47f70bdc879a7a84fb472b822d3ad3158aeef40698d2ce36 as base
```

Comment:
I am using node instead of node-alpine because it is recommended https://github.com/nodejs/docker-node#image-variants. Basically alpine might lack some functionalities can cause small problems/overheads in our work.